plugins/vpp/natplugin/vppcalls/vpp1908/dump_nat_vppcalls_test.go

```diff
@@ -0,0 +1,379 @@
+	Expect(dnat.StMappings[0].ExternalPort).To(BeEquivalentTo(80))
+	Expect(dnat.StMappings[0].LocalIps).To(HaveLen(1))
+	Expect(dnat.StMappings[0].LocalIps[0].VrfId).To(BeEquivalentTo(1))
+	Expect(dnat.StMappings[0].LocalIps[0].LocalIp).To(Equal("10.10.11.140"))
+	Expect(dnat.StMappings[0].LocalIps[0].LocalPort).To(BeEquivalentTo(8081))
+	Expect(dnat.StMappings[0].LocalIps[0].Probability).To(BeEquivalentTo(0))
+	// -> LB mapping
+	Expect(dnat.StMappings[1].TwiceNat).To(Equal(vpp_nat.DNat44_StaticMapping_DISABLED))
+	Expect(dnat.StMappings[1].Protocol).To(Equal(vpp_nat.DNat44_UDP))
+	Expect(dnat.StMappings[1].ExternalInterface).To(BeEmpty())
+	Expect(dnat.StMappings[1].ExternalIp).To(Equal("10.36.20.60"))
+	Expect(dnat.StMappings[1].ExternalPort).To(BeEquivalentTo(53))
+	Expect(dnat.StMappings[1].LocalIps).To(HaveLen(2))
+	Expect(dnat.StMappings[1].LocalIps[0].VrfId).To(BeEquivalentTo(0))
+	Expect(dnat.StMappings[1].LocalIps[0].LocalIp).To(Equal("10.10.11.161"))
+	Expect(dnat.StMappings[1].LocalIps[0].LocalPort).To(BeEquivalentTo(53))
+	Expect(dnat.StMappings[1].LocalIps[0].Probability).To(BeEquivalentTo(1))
+	Expect(dnat.StMappings[1].LocalIps[1].VrfId).To(BeEquivalentTo(0))
+	Expect(dnat.StMappings[1].LocalIps[1].LocalIp).To(Equal("10.10.11.162"))
+	Expect(dnat.StMappings[1].LocalIps[1].LocalPort).To(BeEquivalentTo(153))
+	Expect(dnat.StMappings[1].LocalIps[1].Probability).To(BeEquivalentTo(2))
+
+	dnat = dnats[2]
+	Expect(dnat.Label).To(Equal("DNAT 3"))
+	Expect(dnat.StMappings).To(HaveLen(0))
+	Expect(dnat.IdMappings).To(HaveLen(2))
+	// 1st mapping
+	Expect(dnat.IdMappings[0].VrfId).To(BeEquivalentTo(1))
+	Expect(dnat.IdMappings[0].Protocol).To(Equal(vpp_nat.DNat44_UDP))
+	Expect(dnat.IdMappings[0].Port).To(BeEquivalentTo(0))
+	Expect(dnat.IdMappings[0].IpAddress).To(Equal("10.10.11.200"))
+	Expect(dnat.IdMappings[0].Interface).To(BeEmpty())
+	// 2nd mapping
+	Expect(dnat.IdMappings[1].VrfId).To(BeEquivalentTo(1))
+	Expect(dnat.IdMappings[1].Protocol).To(Equal(vpp_nat.DNat44_UDP))
+	Expect(dnat.IdMappings[1].Port).To(BeEquivalentTo(0))
+	Expect(dnat.IdMappings[1].IpAddress).To(BeEquivalentTo("0.0.0.0"))
+	Expect(dnat.IdMappings[1].Interface).To(BeEquivalentTo("if1"))
+}
+
+func natTestSetup(t *testing.T) (*vppcallmock.TestCtx, vppcalls.NatVppAPI, ifaceidx.IfaceMetadataIndexRW, idxmap.NamedMappingRW) {
+	ctx := vppcallmock.SetupTestCtx(t)
+	log := logrus.NewLogger("test-log")
+	swIfIndexes := ifaceidx.NewIfaceIndex(logrus.DefaultLogger(), "test-sw_if_indexes")
+	dhcpIndexes := idxmap_mem.NewNamedMapping(logrus.DefaultLogger(), "test-dhcp_indexes", nil)
+	natHandler := vpp1908.NewNatVppHandler(ctx.MockChannel, swIfIndexes, dhcpIndexes, log)
+	return ctx, natHandler, swIfIndexes, dhcpIndexes
+}
+func ipTo4Address(ipStr string) (addr bin_api.IP4Address) {
+	netIP := net.ParseIP(ipStr)
+	if ip4 := netIP.To4(); ip4 != nil {
+		var ip4Addr bin_api.IP4Address
+		copy(ip4Addr[:], netIP.To4())
+		addr = ip4Addr
+	} else {
+		return addr
```

Comment:
This case does nothing. Both statements `return` and `return addr` do same thing.